Makefile

```diff
@@ -119,7 +119,7 @@ vet: ## Run go vet against code.
 
 .PHONY: test
 test: manifests generate fmt vet envtest ## Run tests.
-	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" go test ./... --tags=test_all -coverprofile cover.out
+	export SSL_CERT_FILE=${ROOT_DIR}/controllers/testdata/tls/ca-bundle.crt && KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" go test ./... --tags=test_all -coverprofile cover.out
```

Comment:
can you explain why this change is needed? at face value it seems unrelated

Reply:
@HumairAK that's the first commit in this PR. It's unrelated to the rest of the PR but I wanted to add it since it fixes the `make test` command. Otherwise, the CA bundle is my laptop's CA bundle and it causes the functional tests to fail.